### ISSUE TITLE
ES5 property descriptor support is somewhat lacking

### ISSUE THREAD

**Issue by capaj** [mobxjs/mobx#110]
mobservable tries to redefine a property marked as configurable: false into setter/getter
```javascript
var te = {}
Object.defineProperty(te, 'key', {
  enumerable: true,
  configurable: false,
  writable: false,
  value: 'static'
})

const a =  mobservable.observable(te)
// last line throws
/home/capaj/git_projects/mobservable-db/node_modules/mobservable/lib/observableobject.js:53
        Object.defineProperty(this.target, propName, {
               ^

TypeError: Cannot redefine property: key
    at Function.defineProperty (native)
    at ObservableObject.defineReactiveProperty (/home/capaj/git_projects/mobservable-db/node_modules/mobservable/lib/observableobject.js:53:16)
    at ObservableObject.set (/home/capaj/git_projects/mobservable-db/node_modules/mobservable/lib/observableobject.js:38:18)
    at extendObservableHelper (/home/capaj/git_projects/mobservable-db/node_modules/mobservable/lib/core.js:348:18)
    at makeChildObservable (/home/capaj/git_projects/mobservable-db/node_modules/mobservable/lib/core.js:440:16)
    at Object.observable (/home/capaj/git_projects/mobservable-db/node_modules/mobservable/lib/core.js:43:20)
    at Object.<anonymous> (/home/capaj/git_projects/mobservable-db/aa.js:11:24)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)

```

I think when a property is not configurable, mobservable should just skip it.

Also when property is `configurable: true` but marked as `writable: false`, mobservable should also skip it and do not try to make it into a getter/setter.

This should be easily done with like 2 if statements(+ 2 simple tests)-should I bother with a PR, or can you do it yourself @mweststrate ?

**Comment by mweststrate** [comment]
Fixed in the feature/2 branch, will be released soon.

**Comment by mweststrate** [comment]
Released as part of `MobX@2.0.0`
